docs/ingestion/kafka-ingestion.md

```diff
@@ -134,6 +134,12 @@ If you enable multi-topic ingestion for a datasource, downgrading to a version o
 28.0.0 will cause the ingestion for that datasource to fail.
 :::
 
+:::info
+Migrating an existing supervisor to use `topicPattern` instead of `topic` is not supported. It is also not supported to change the `topicPattern` of an existing supervisor to a different regex pattern.
```

Comment:
Thanks for adding this.

Maybe you can put it in a list format for readability like 
```
The following updates to a supervisor spec are not supported:
- item 1
- item 2
- item 3
```

Reply:
Updated with list

Reply:
Thanks!